core/src/main/java/com/redhat/lightblue/client/request/AbstractLightblueMetadataRequest.java

```diff
@@ -4,30 +4,48 @@
 
 import com.fasterxml.jackson.databind.JsonNode;
 
-public abstract class AbstractLightblueMetadataRequest extends AbstractLightblueRequest {
+import com.redhat.lightblue.client.http.HttpMethod;
 
-    public AbstractLightblueMetadataRequest(String entityName, String entityVersion) {
-        super(entityName, entityVersion);
+public abstract class AbstractLightblueMetadataRequest extends LightblueRequest {
```

Comment:
Changed now,

On Fri, May 20, 2016 at 12:20 PM, Dennis Crissman <notifications@github.com>
wrote:

> In
> core/src/main/java/com/redhat/lightblue/client/request/AbstractLightblueMetadataRequest.java
> <https://github.com/lightblue-platform/lightblue-client/pull/293#discussion_r64085222>
> :
>
> >
> > -    public AbstractLightblueMetadataRequest(String entityName, String entityVersion) {
> > -        super(entityName, entityVersion);
> > +public abstract class AbstractLightblueMetadataRequest extends LightblueRequest {
>
> Why is this class's name not changed?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/lightblue-platform/lightblue-client/pull/293/files/a26850a67fc0252d7f24a09ecf4c45fd80b40b2f#r64085222>
>